leader/leader.go

```diff
@@ -167,6 +167,7 @@ func Become(ctx context.Context, lockName string, opts ...Option) error {
 				log.Info("Leader lock configmap owner reference must be a pod.", "OwnerReference", existingOwners[0])
```

Comment:
We need to add the logic of the fragments to here as well for we are able to generate the changelog. 

c/c @jmrodri

Reply:
There's really no changelog fragment for the operator-lib. The releases are just a signed tag.

Reply:
IMO we should have since we should have a way to know what changed in each version/release.